Formula/libxml2.rb

```diff
@@ -30,24 +38,19 @@ class Libxml2 < Formula
     sha256 "1ec98ced07f46c0976bd919fe97cf72092a28c1338d43410384d37f6439634c0" => :yosemite
   end
 
-  head do
-    url "https://git.gnome.org/browse/libxml2.git"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
-
-  keg_only :provided_by_osx
+  keg_only :provided_by_macos
 
   depends_on :python if MacOS.version <= :snow_leopard
 
-  def install
-    if build.head?
-      inreplace "autogen.sh", "libtoolize", "glibtoolize"
-      system "./autogen.sh"
-    end
+  # These should return to being head-only whenever 2.9.5 is released.
+  depends_on "pkg-config" => :build
```

Comment:
Aye. Also wasn't a problem locally with my newer `zlib` on High Sierra, only stumbled across the failure on CI. I do have `pkg-config` installed, but it shouldn't have found it given it's not a dependency. Maybe an old `zlib` bug or something, it's a strange one.